src/js/index.js

```diff
@@ -28,8 +28,34 @@ $('.responsive').slick({
 			slidesToScroll: 1
 		}
 	}
-  // You can unslick at a given breakpoint now by adding:
-  // settings: "unslick"
-  // instead of a settings object
-	]
-});
\ No newline at end of file
+		// You can unslick at a given breakpoint now by adding:
+		// settings: "unslick"
+		// instead of a settings object
+		]
+});
+
+//---------------Show/Hide FAQs---------------//
+function showonlyone(thechosenone) {
+	var newboxes = document.getElementsByTagName("div");
```

Comment:
Seems aggressive to get all the `div`s in the entire document. Could you scope this back so you are only needing to loop through the proper elements?